normalize-git-url.js

```diff
@@ -23,8 +23,10 @@ module.exports = function normalize (u) {
   if (altered) {
     if (u.match(/^git\+https?/) && parsed.pathname.match(/\/?:[^0-9]/)) {
       returnedUrl = u.replace(/^git\+(.*:[^:]+):(.*)/, '$1/$2')
+    } else if (u.match(/^git\+file/)) {
+      returnedUrl = u.replace(/^git\+/, '')
     } else {
-      returnedUrl = u.replace(/^(?:git\+)?ssh:\/\//, '')
+      returnedUrl = u.replace(/^(?:git\+)?ssh:[\/\\]{2}/, '')
```

Comment:
I was going to ask under what circumstances an ssh url would end up as `ssh:\\` but I guess it doesn't really matter. =p